### DISCUSSION TITLE
Image attachment on pdf.js

### DISCUSSION THREAD

**Discussion by kamaludin21** [mozilla/pdf.js#17783] [Q&A]
Hello, 

i want use feature add image on pdf file like this demo: **https://mozilla.github.io/pdf.js/web/viewer.html**

any example repo or blog tutorial ?

**Comment by lasseborly** [comment]
Not really. You have to dig through the source code I am afraid. It's quite complex. You are better of just using the web viewer as is and tailoring your implementation around that. Disabling the features you do not want.

**Reply by kamaludin21** [reply]
Thanks buddy
